parquet-encoding/src/main/java/org/apache/parquet/bytes/BytesInput.java

```diff
@@ -164,6 +176,9 @@ public static BytesInput copy(BytesInput bytesInput) throws IOException {
     return baos.getBuf();
   }
 
+  public ByteBuffer toByteBuffer() throws IOException {
+    return ByteBuffer.wrap(toByteArray());
+  }
```

Comment:
Added methods to a few of the subclasses to override the behavior and avoid a copy or extra method call where possible.